java/bulk-import/bulk-import-starter/src/main/java/sleeper/bulkimport/starter/BulkImportStarterLambda.java

```diff
@@ -102,28 +95,18 @@ public Void handleRequest(SQSEvent event, Context context) {
         propertiesReloader.reloadIfNeeded();
         event.getRecords().stream()
                 .map(SQSEvent.SQSMessage::getBody)
-                .map(message -> deserialiseAndValidate(message, new BulkImportJobSerDe()::fromJson,
-                        job -> job.toIngestJob().getValidationFailures(), ingestJobStatusStore,
-                        invalidJobIdSupplier, timeSupplier))
+                .map(message -> ingestJobValidationUtils.deserialiseAndValidate(message,
+                        new BulkImportJobSerDe()::fromJson, job -> job.toIngestJob().getValidationFailures()))
                 .filter(Optional::isPresent)
                 .map(Optional::get)
-                .map(this::expandDirectories)
+                .map(job -> ingestJobValidationUtils.expandDirectories(job.toIngestJob(),
+                        files -> job.toBuilder().files(files).build()))
```

Comment:
This actually breaks if the bulk import job doesn't have an ID, because it reports a validation failure to the status store against the ID, and the ID doesn't get set until it gets to the BulkImportExecutor. This needs standardising.